### ISSUE TITLE
Request for default adoption of Ctrl + Tab for tab navigation

### ISSUE THREAD

**Issue by alexei** [zed-industries/zed#7321]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

I have found that Zed employs Ctrl + - as a keyboard shortcut for tab navigation, which diverges from the [widely recognized](https://en.wikipedia.org/wiki/Table_of_keyboard_shortcuts#Tab_management) Ctrl + Tab combination. The current choice is unconventional and [unexpected](https://en.wikipedia.org/wiki/Principle_of_least_astonishment), thus I would like to suggest considering the adoption of the standard Ctrl + Tab shortcut by default, for convenience and a more intuitive experience.

### Environment

Zed: v0.120.6 (Zed)

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by alber70g** [comment]
`CTRL` + `Tab` is different from `CTRL` + `-` and it's reverse counterpart `CTRL` + `SHIFT` + `-`

You're referring to pane switching, either in the "alt-tab"-way where you switch between last active window. The current implementation is more akin "back" and "forward" buttons of a browser, and is on-par with other implementations, at least in VSCode.

**Comment by alexei** [comment]
@alber70g I'm not sure how to read your comment, but I feel I need to clarify one thing: by "tab navigation" I mean going to the next/previous tab. In a web browser that would be switching between tabs, as opposed to navigating through history. I might have used the wrong terminology without realising it might lead to confusion.

As a side note, Ctrl + Tab and Ctrl + - both seem to be doing the same thing in VSCode: switching between tabs. I didn't even know about Ctrl + - until Zed...

**Comment by yuk0ga** [comment]
@alexei 
I was able to configure this by adding ↓ to my `keymap.json`
```json
[
  {
    "context": "Editor",
    "bindings": {
      "ctrl-shift-tab": "pane::ActivatePrevItem",
      "ctrl-tab": "pane::ActivateNextItem"
    }
  }
]
```

**Comment by garth** [comment]
The keymap bindings suggested by @yuk0ga allow you to move to the tab to the right or tab to the left of the current tab. This is not what other editors do. `ctrl+tab` should take you back in history and `ctrl+shift+tab` should take you forward in history (MRU - Most Rencently Used).

Unless you only have 2 tabs open or you like to manually re-order your tabs regularly, moving between tabs to the left or right is not so useful.

**Comment by AlessandroVol23** [comment]
This is my main dealbreaker as well since I'm so used to it. 



> @alexei I was able to configure this by adding ↓ to my `keymap.json`
> 
> ```json
> [
>   {
>     "context": "Editor",
>     "bindings": {
>       "ctrl-shift-tab": "pane::ActivatePrevItem",
>       "ctrl-tab": "pane::ActivateNextItem"
>     }
>   }
> ]
> ```

This only selects the next and previous ones but not the latest active ones. I can't find the option. Do you know if it is possible to get the same behaviour as in VS Code as well?

**Comment by martin-braun** [comment]
I also need a way to move tabs to the left or right, something like `pane::MoveToLeft` and `pane::MoveToRight`.

**Comment by lishine** [comment]
This is a deal braker.
ctrl-tab should take me to the previous active tab
I don't use ctrl-shift-tab

not only ctrl-tab does not work and cannot be configures in cycle mode,
as soon as I select file in the files panel, no keyboard shortcut works for switching tabs!

Why ruin the experience of such a fast editor  ?

**Comment by hedefalk** [comment]
Plus one on dealbreaker unfortunately. This is the way I'm used to navigating tabs. I actually do not want to show tabs at all, but always just use ctrl-tab for navigating by MRU. It's such muscle memory now that I cannot use an editor without it. Or maybe I'll try if I can do with: 

```
      "ctrl-tab": "pane::ActivateLastItem",
      "alt-cmd-left": "pane::GoBack",
      "alt-cmd-right": "pane::GoForward"
```

for popping one item in the MRU-stack with ctrl-tab and resolve to arrow navigation if I need to go further.

**Comment by miko007** [comment]
i am currently searching for a solution for the same issue and landed here. the core problem seems to be, that zed does not even seem to have a concept of a "active tab history".

it basically is the list in the <kbd>⌘ + P</kbd> file search *(called `file_finder` in the default keymap)*, so we would need an action, that brings the tab of the last item *(the top one)* on that list to the front, if that file is currently open.

what vscode does, is opening the list, selecting the last item and bringing it to the front on release of the <kbd>ctrl</kbd> key. when the <kbd>ctrl</kbd> is not released, but <kbd>tab</kbd> is pressed again, it moves down the list by one item.

this also is the behaviour most people would expect, i assume. <kbd>ctrl + shift + tab</kbd> does the same thing, but starting from the bottom.

in conclusion, we would need a new action, that:

* opens the `file_finder` on press of a key combination
* selects the most recent item
* selects the next item in that list, as long as the modifier key is still pressed
* brings the corresponding tab to the foreground on release of the modifier key

and additionally an action, that does the same, but from the bottom up.

**Comment by neel04** [comment]
+1 its pretty annoying. What I feel users want is:

- Cmd/Ctrl + Tab ==> Switch to last active tab. Pressing the shortcut again should bring to back to the OG tab
- Cmd + 1/2/3/4/... ===> Switch to particular tab

This should be the default behavior IMO, especially if you want ppl to ditch Vscode

**Comment by anuragg-p** [comment]
Hi can someone pls point out how to do this with vim mode?
 ``` {
    "context": "Pane",
    "bindings": {
      "shift h": "pane::ActivatePrevItem",
      "shift l": "pane::ActivateNextItem"
    }
  }
```
The above doesn't seem to work with vim mode.
Thanks

**Comment by elitan** [comment]
Any way to configure:

> Cmd + 1/2/3/4/... ===> Switch to particular tab

which is what I'm using in Chrome, Cursor, and iTerm?

**Comment by vinayakkulkarni** [comment]
> Any way to configure:
> 
> > Cmd + 1/2/3/4/... ===> Switch to particular tab
> 
> which is what I'm using in Chrome, Cursor, and iTerm?


```json
[
  {
    "context": "Editor",
    "bindings": {
      "ctrl-shift-f": "editor::Format",
      "cmd-k-s": "workspace::SaveWithoutFormat"
    }
  },
  {
    "context": "Pane",
    "bindings": {
      "cmd-1": ["pane::ActivateItem", 0],
      "cmd-2": ["pane::ActivateItem", 1],
      "cmd-3": ["pane::ActivateItem", 2],
      "cmd-4": ["pane::ActivateItem", 3],
      "cmd-5": ["pane::ActivateItem", 4],
      "cmd-6": ["pane::ActivateItem", 5],
      "cmd-7": ["pane::ActivateItem", 6],
      "cmd-8": ["pane::ActivateItem", 7],
      "cmd-9": ["pane::ActivateItem", 8],
      "cmd-0": "pane::ActivateLastItem"
    }
  }
]
```

This was working before but now it's gone to shit, idk what got changed :X

**Comment by BBlackwo** [comment]
FYI the binding has changed from `pane::ActivatePrevItem` to `pane::ActivatePreviousItem`

```jsonc
[
  {
    "context": "Editor",
    "bindings": {
      "ctrl-shift-tab": "pane::ActivatePreviousItem", // Changed from pane::ActivatePrevItem
      "ctrl-tab": "pane::ActivateNextItem"
    }
  }
]
```
